Capstone Project/ViewControllers/Stats/StatsViewController.m

```diff
@@ -351,10 +347,8 @@ -(NSMutableArray *)logarithmicRegression:(NSMutableArray *)Y{
     
     NSMutableArray *logarithmicTrendline = [[NSMutableArray alloc] init];
     
-    for(int i = 0; i < (X.count + 3); i++){
-        
-//        prediction = A + B * log([X[i] doubleValue]);
-        prediction = A + B * log(i + 1);
+    for(int i = 1; i < ((n+1) + 3); i++){
```

Comment:
Very cool, so if I understand log regression its just y = m*log(x) + b as opposed to y = mx + b?

Reply:
that's right